config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  devise_for :users
+  get 'welcome/index'
+
   # The priority is based upon order of creation: first created -> highest priority.
```

Comment:
@snugthejoiner let's keep a clean routes file. Please delete all of the comments out of it.